src/main/java/at/laborg/briss/cli/BrissCMD.java

```diff
@@ -74,23 +71,26 @@ public static void autoCrop(final String[] args) {
         System.out.println("Calculating crop rectangles.");
         try {
             for (PageCluster cluster : clusterDefinition.getClusterList()) {
-                float[] auto = CropFinder.getAutoCropFloats(cluster.getImageData().getPreviewImage());
+                float[] auto =
+                        CropFinder.getAutoCropFloats(cluster.getImageData().getPreviewImage());
```

Comment:
Is there a way to set the max width for the formatter? I would like to set it to 120!
In this case the line only has 103 characters, no IMO no need to wrap it already as screens are rather wide these days 😃

Reply:
I think there is. I'll check that out and set to 120. But the issue is not about the screen width but the amount of movement the reader' eyes will have to make in order to read the entire line.

Reply:
No luck. With Google/Palantir the max line length can't be changed. :(